packages/transformers/js/src/JSTransformer.js

```diff
@@ -126,6 +126,37 @@ const SCRIPT_ERRORS = {
   },
 };
 
+function convertAliasReactIntoPragma(
```

Comment:
In order to convert `preact/compat` into `preact` pragma, I made a function to do it, but I feel this approach is a little muddy. If you have another idea to implement this, I will be happy if you comment on it.